drivers/Makefile.dep

```diff
@@ -43,6 +43,10 @@ ifneq (,$(filter at,$(USEMODULE)))
   USEMODULE += fmt
   USEMODULE += isrpipe
   USEMODULE += isrpipe_read_timeout
+  ifneq (,$(filter at_urc_isr,$(USEMODULE)))
+    USEMODULE += event_thread_medium
```

Comment:
why medium ?

Reply:
It seemed to me that this priority level was a good choice for a driver. But I guess we could make it configurable. I can think on having `at_urc_isr_<prio>` instead, that way we could pull in the needed module.

Reply:
Changed